### PR TITLE
feat(README): organized links in Repo

### PR DESCRIPTION
**Pull request by NicAtak**
## Changes
Organized links in the README

## Purpose
Easier to navigate and read

## Approach
Anju Driver Nicole Navigator

Closes #00

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,7 @@
 
 
 
-* Error: Permission denied (publickey info
-https://docs.github.com/en/authentication/troubleshooting-ssh/error-permission-denied-publickey
+
```
Comment (michaelachrisco) [inline README.md:9]: Why was this link removed?

File: README.md [hunk 2]
```diff
@@ -19,24 +18,27 @@ https://docs.github.com/en/authentication/troubleshooting-ssh/error-permission-d
...
 
-#### [DevOps roadmap and resource link](https://github.com/annfelix/DEVOPS-WORLD)
-#### [CLI Tools](https://dev.to/lissy93/cli-tools-you-cant-live-without-57f6)
-#### [Chrome-Extension for job seeker](https://www.formpl.us/blog/job-chrome-extensions#:~:text=Formplus%20is%20a%20great%20tool,your%20responses%20for%20later%20use)
 
 
+
+
+## Git/Github Helpful Links
+
```
Comment (michaelachrisco) [inline README.md]: Do we need those extra return statements?

File: README.md [hunk 3]
```diff
@@ -68,10 +63,13 @@ https://docs.github.com/en/authentication/troubleshooting-ssh/error-permission-d
 - [From Hashicorp](https://www.youtube.com/watch?v=YcJ9IeukJL8)
 - [Freecodecamp](https://www.youtube.com/watch?v=SLB_c_ayRMo)
 - [8 Best Pratices](https://www.youtube.com/watch?v=gxPykhPxRW0)
+- [Learn terraform ](https://youtu.be/iRaai1IBlB0)
```
Comment (michaelachrisco) [inline README.md]: nitpick but I believe this should be 
```suggestion
- [Learn terraform](https://youtu.be/iRaai1IBlB0)
```
without the space.

**Review by michaelachrisco — APPROVED** [review]
Looks good @NicAtak ! I like the new previewed README. Looks much more structured. 

Added a couple of comments. The only issue I see is with these big PRs where we lint a large file(s), it may break other peoples PRs. The best practice is to make PRs as small as possible. Its easier to review for everyone involved if the PRs are small and there is less likely going to be merge conflicts for everyone else.

File: README.md (hunk 1 above)
Comment (NicAtak) [reply README.md:9]: I removed it from this location and added it to line 102

File: README.md (hunk 3 above)
Comment (NicAtak) [reply README.md]: I like nitpicky! I want ALL the notes and suggestions! It's the only way to learn and grow.

**Comment by NicAtak** [comment]
Re: "The only issue I see is with these big PRs where we lint a large file(s), it may break other peoples PRs. The best practice is to make PRs as small as possible. Its easier to review for everyone involved if the PRs are small and there is less likely going to be merge conflicts for everyone else."
how should I go about making these types of changes in the future? Should I still got through with the pull request?

**Review by michaelachrisco — APPROVED** [review]
Looks good! Nice job.

**Pull request merged**